aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageUploadFileOperation.java

```diff
@@ -101,7 +104,7 @@ public void onStateChanged(int transferId, TransferState state) {
                     @SuppressWarnings("checkstyle:MagicNumber")
                     @Override
                     public void onProgressChanged(int transferId, long bytesCurrent, long bytesTotal) {
-                        int percentage = (int) (bytesCurrent / bytesTotal * 100);
+                        // int percentage = (int) (bytesCurrent / bytesTotal * 100);
```

Comment:
Should we remove this for now?

Reply:
Will update to leave the line as it had been originally. Maybe it shouldn't be here, but it's not really the point of this current PR.